### PR TITLE
NEXUS-5192: Implementing the property to control "local avail check"

### PR DESCRIPTION
**Pull request by cstamas**
Making it default to false (reverse behaviour as so far), but still
making it possible to be enabled if needed.

This change reverts how NX behaved so far.

CI:
https://builds.sonatype.org/view/nexus-features/job/nexus-oss-its-feature/423/

### CODE REVIEW

**Comment by cstamas** [comment]
CI +1'd
(but both slave jobs failed as CompositeExceptionTest from master was missing license header, fixed in b4bfb187ac48e0a2fb0e3856c3e19f7a0f5764bb)

**Comment by ifedorenko** [comment]
+1 

code format looks proper in the diff

File: nexus/nexus-core-plugins/nexus-indexer-lucene-plugin-parent/nexus-indexer-lucene-plugin/src/main/java/org/sonatype/nexus/index/treeview/DefaultMergedTreeNodeFactory.java [hunk 1]
```diff
@@ -159,18 +118,65 @@ protected ResourceStoreRequest getResourceStoreRequest( String path )
...
+                }
+            }
+        }
+        catch ( ItemNotFoundException e )
+        {
+            // mute it, probaly not available locally
+        }
+        catch ( Exception e )
+        {
+            // TODO: do something here?
```
Comment (nabcos) [inline nexus/nexus-core-plugins/nexus-indexer-lucene-plugin-parent/nexus-indexer-lucene-plugin/src/main/java/org/sonatype/nexus/index/treeview/DefaultMergedTreeNodeFactory.java]: at least some logging?

**Comment by adreghiciu** [comment]
+1

**Comment by nabcos** [comment]
+1 but maybe handle the TODO? It's only for UI though, so ignoring it may be fine, but then the comment should say so.

**Pull request merged**